### PR TITLE
zeros

### PR DESCRIPTION
**Pull request by Guru-mp3**

### CODE REVIEW

**Comment by jkeane508** [comment]
Hi

Can you please specify which issue this PR resolves?

Thanks.

**Comment by Guru-mp3** [comment]
zeros #7603

**Comment by Guru-mp3** [comment]
@jkeane508 Any updates or any changes required?

File: ivy/functional/ivy/creation.py [hunk 1]
```diff
@@ -331,6 +331,9 @@ def zeros(
            [0., 0., 0., 0., 0.],
            [0., 0., 0., 0., 0.]])
 
+    >>> x = ivy.zeros(5)
```
Comment (jkeane508) [inline ivy/functional/ivy/creation.py:334]: There is already an example of how this function works above. Not sure this example is needed.

File: ivy/functional/backends/numpy/creation.py [hunk 2]
```diff
@@ -214,6 +214,9 @@ def zeros(
     return _to_device(np.zeros(shape, dtype), device=device)
 
 
+zeros.support_native_out = True
```
Comment (jkeane508) [inline ivy/functional/backends/numpy/creation.py:217]: What was your reasoning for adding this line?

**Review by jkeane508 — CHANGES_REQUESTED** [review]
Hi, thanks for submitting this PR

Just have a few questions/changes :)

File: ivy/functional/backends/numpy/creation.py (hunk 2 above)
Comment (Guru-mp3) [reply ivy/functional/backends/numpy/creation.py:217]: I had added this line as this is a numpy primary function and it supports out argument. I referred the Inplace Updates section of the Deep Dive.

File: ivy/functional/ivy/creation.py (hunk 1 above)
Comment (Guru-mp3) [reply ivy/functional/ivy/creation.py:334]: I added this as it is mentioned in Docstring Examples to cover all possible variants. There was one example which mentioned a tuple. Hence, I added a number

**Pull request merged**